src/molecules/FeedbackPage/index.tsx

```diff
@@ -0,0 +1,167 @@
+      if (star == 0) {
+        toast.error(`Please provide valid review`);
+      } else {
+        setTimeout(() => {
+          toast.success(`Review sent successfully`);
+          setReview("");
+        }, 2000);
+      }
+    } else if (rateBox && !reviewContainer) {
+      if (star === 0) {
+        toast.error(`Please provide valid review`);
+      } else {
+        setTimeout(() => {
+          toast.success(`Review sent successfully`);
+        }, 2000);
+      }
+    } else if (!rateBox && reviewContainer) {
+      if (review === "") {
+        toast.error(`Please provide valid review`);
+      } else {
+        setTimeout(() => {
+          toast.success(`Review sent successfully`);
+          setReview("");
+        }, 2000);
+      }
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+
+      <Typography
+              sx={{
+                fontSize: "5vh",
+                fontWeight: 'bold',
+                m: 2,
+                p: 2,
+                display: 'fixed'
+              }}
+            >
+        {config.component.Title}
+      </Typography>
+
+      <Box className={styles.main}>
+        {config.component.ratingBox === true ? (
+          <Box className={styles.section}>
+            <Typography
+              sx={{
+                fontWeight: "bold",
+                fontSize: "3vh",
+              }}
+            >
+              {config.component.ratingBoxTitle}
+            </Typography>
+
+            <Rating
+              data-testid= "ratingComponent"
+              name="simple-controlled"
+              value={star}
+              max={config.component.ratingMaxStars}
+              
+              // @ts-ignore
+              onChange={(event, newValue) => {
+                setStar(() => {
+                  return newValue === null ? 1 : newValue;
+                });
+              }}
+              defaultValue={1}
+              sx={{
+                fontSize: '6vh'
+              }}
+            />
+            <Typography
+              sx={{
+                textAlign: "center",
+                fontSize: "2vh"
+              }}
+            >
+              {config.component.ratingStarDescription}
+            </Typography>
+            <Button
+              id="ratingBtn"
+              variant="contained"
+              data-testid="ratingBtn"
+              sx={{
+                mt: 2,
+                backgroundColor: `${config.theme.primaryColor.value}`,
+                fontWeight: "bold",
+                borderRadius: "10rem",
+                fontSize: "10px",
+                p: 1.5,
+                "&:hover": {
+                  backgroundColor: `${config.theme.secondaryColor.value}`,
+                },
+              }}
+              onClick={handleFeedback}
+            >
+              {config.component.ratingButtonText}
+            </Button>
+          </Box>
+        ) : null}
+
+        {config.component.reviewBox === true ? (
```

Comment:
since you are rendering null for else case ,then it's better to do it like 
`{condition && <Component />} `